manifests/i/ima/menu/1.0.0/ima.menu.yaml

```diff
@@ -0,0 +1,18 @@
+PackageIdentifier: ima.menu
+PackageVersion: 1.0.0
+Publisher: ima
+PackageName: menu
+PackageLocale: en-US
+License: Proprietary
+ShortDescription: A modern context menu for Windows.
+Installers:
+  - Architecture: x64
+    InstallerType: nullsoft
+    InstallerUrl: https://github.com/iMAboud/iMA-Menu-Plugins/releases/download/1.0.0/iMA.Menu.exe
+    InstallerSha256: eecc4a4c8a890eea2d3e3a132cf64b9b8c904a8ce210cbde10ac94c6dacb7b89
+    UpgradeBehavior: install
+    InstallerSwitches:
+      Silent: /S
+      SilentWithProgress: /S
+ManifestType: singleton
```

Comment:
Hi @iMAboud,

Please be aware that the [singleton manifest format has been depreciated](https://github.com/microsoft/winget-pkgs/issues/181590), and feel free to utilize the [latest multi-file format](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.10.0).